python/tvm/contrib/rocm.py

```diff
@@ -168,3 +168,34 @@ def callback_rocm_bitcode_path(rocdl_dir=None):
             raise RuntimeError("could not find bitcode " + n)
 
     return tvm.runtime.convert(bitcode_files)
+
+
+def parse_compute_version(compute_version):
+    """Parse compute capability string to divide major and minor version
+
+    Parameters
+    ----------
+    compute_version : str
+        GFX version of a GPU (e.g. "6.0")
+
+    Returns
+    -------
+    major : int
+        major version number
+    minor : int
+        minor version number
+    """
+    split_ver = compute_version.split(".")
+    try:
+        major = int(split_ver[0])
+        minor = int(split_ver[1])
+        return major, minor
+    except (IndexError, ValueError) as err:
+        raise RuntimeError("Compute version parsing error: " + str(err))
+
+
+def support_mfma(device):
+    major, minor = parse_compute_version(device.compute_version)
+    if major >= 9 and minor >= 8:
```

Comment:
Since probably mfma is only available on gfx908, how about directly comparing against "908"? Newer arch like navi2 is gfx1030 but it does not have mfma.